avocado/selftests/unit/utils/test_archive.py

```diff
@@ -0,0 +1,572 @@
+import hashlib
```

Comment:
IIUIC the path to this file is a mistake. All avocado tests should be under `selftests` not `avocado/selftests` dir. Also, I think those tests should be part of  `selftests/unit/utils/archive.py` file.